src/components/React.js

```diff
@@ -3,7 +3,16 @@ class React {
      * Required dependencies for the component.
      */
     dependencies() {
-        return ['@babel/preset-react'];
+        const dependencies = ['@babel/preset-react'];
+
+        if (Mix.isHot()) {
+            dependencies.concat([
+                '@pmmmwh/react-refresh-webpack-plugin',
+                'react-refresh'
+            ]);
```

Comment:
I'm inclined to say we should always include these.

Reply:
Why? I think this is only necessary when using HMR.

Reply:
Because once they're added to the package.json file they'll be in there whether you're using it or not. It's fine either way honestly I just think it'd be less hassle.

Reply:
![image](https://user-images.githubusercontent.com/22997803/101367058-61ba5b00-38ae-11eb-88d6-6cde24392d1b.png)